src/app/dashboard/competitions/page.tsx

```diff
@@ -0,0 +1,29 @@
+import { getServerSession } from "next-auth";
+import { redirect } from "next/navigation";
+import { authOptions } from "@/lib/auth";
+import CompetitionsList from "./components/CompetitionsList";
+import { getCompetitionsForOrganizer } from "@/lib/db/competition";
+
+export default async function CompetitionsPage() {
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.organizerId) {
+    redirect("/login");
+  }
+
+  const competitions = await getCompetitionsForOrganizer(session.user.organizerId);
```

Comment:
Tratemos de ser lo más claros con los llamados a DB. Por puro mañosa lo cambiaría a `getCompetitionsForOrganizerId`

```suggestion
  const competitions = await getCompetitionsForOrganizerId(session.user.organizerId);
```

Reply:
Me parece perfecto, muy de acuerdo en ser lo más descriptivos que se pueda. 

Pd: acostumbrémonos a que el que inicia el comentario/sugerencia la de por resuelta (marcando el botón de "Resolver conversation")